includes/admin/class-wcmypa-admin.php

```diff
@@ -995,9 +996,12 @@ private function getConfirmationData(DeliveryOptions $deliveryOptions): ?array
             ];
         }
 
-        $confirmationData = [
-            __("delivery_type", "woocommerce-myparcel") => WCMP_Data::getDeliveryTypesHuman()[$deliveryOptions->getDeliveryType()],
-        ];
+        if (WCMYPA()->setting_collection->isEnabled(WCMYPA_Settings::SETTING_DELIVERY_OPTIONS_ENABLED)) {
```

Comment:
Zou je `WCMYPA()->setting_collection->isEnabled(WCMYPA_Settings::SETTING_DELIVERY_OPTIONS_ENABLED)` in een variable kunnen zetten. Dan zal de If duidelijker worden en is het sneller te lezen.

Reply:
done